core/jazz_ui/e2e/04-jazzservices/jazzservices.e2e-spec.ts

```diff
@@ -88,9 +88,91 @@ beforeAll(() => {
         expect(jazzServices_po.getWebsiteServiceName().getText()).toEqual(servicename);
         expect(jazzServices_po.getWebsiteType().getText()).toEqual('website');
         expect(jazzServices_po.getWebsiteStatus().getText()).toEqual('creation started');
-    });
-   
-    });
+  });
+ it('Verify API Service Page Title', () => {
+       browser.wait(EC.visibilityOf(jazzServices_po.getAPIServiceName()), timeOutHigh);
+       browser.wait(EC.elementToBeClickable(jazzServices_po.getAPIServiceName()), timeOutHigh);
+       jazzServices_po.getAPIServiceName().click();
+       browser.wait(EC.visibilityOf(jazzServices_po.getServiceNameHeader()), timeOutHigh);
+       browser.sleep(45000);
+       browser.refresh();
+       browser.driver.switchTo().activeElement();
+       jazzServices_po.getRefresh().click();
+       browser.sleep(30000);
+  });
+  it('Verify API Prod Service', () => {
```

Comment:
please add new lines after every it block(every where)

Reply:
ok

Reply:
Yes